### ISSUE TITLE
Rewrite the contrast toggles in plain HTML

### ISSUE THREAD

**Issue by SavvasStephanides** [open-sdg/open-sdg#755]
Currently the high contrast toggles are being [built dynamically with Javascript](https://github.com/open-sdg/open-sdg/blob/1b7e787c736e8516cad1cd1ea8e48994b28b29af/_includes/assets/js/accessibility.js). This has worked for the most part but it can cause (and has caused) quite a few issues:

* if there's a Javascript bug anywhere in the site, it can cause the rest of the script to crash. This can result in the contrast switcher to not show up at all.
* Building the toggles in Javascript makes them unstable, such as the case of the [toggle disappearing completely in IE11](https://github.com/open-sdg/open-sdg/issues/749). Rewriting the toggles in basic HTML (and just leave the functionality to Javascript only) will make them a lot more stable and lot more compatible (the more basic the better). 
* Overall good practice to have as much markup as possible where it belongs: HTML

I guess my question at this point that I'm trying to figure out is: why was this built with Javascript to begin with? Is there some kind of feature that we can't achieve with basic HTML, or just couldn't at the time of developing?

Nonetheless, I think we will benefit greatly with a rewrite.

**Comment by brockfanning** [comment]
My guess is that because it was done in javascript mainly because it uses cookies to remember the user's choice on the next page load. It also needs to change some image src attributes. I'd bet that we could get it more in HTML and less javascript, but I don't think we'll be able to get to 100% HTML.

**Comment by SavvasStephanides** [comment]
@brockfanning My suggestion is to move as much of the layout logic as possible to HTML/CSS. I understand there's some dynamic Javascript functionality going on which is necessary, such as getting and setting the contrast cookie. What I am suggesting is the basic layout of the toggles to be made in HTML and leave the cookie/toggling logic to Javascript.

It is also common practice to to make dynamic page styling changes using CSS, using attributes:

```scss
#container{
    background-color: #FFFFFF;
}

body.contrast-high{
    #container{
        background-color: #000000;
    }
}
```

We can then use Javascript to toggle the body class name and CSS will do the rest.

```js
document.body.classList.add("contrast-high")
```

And obviously yes, we will need Javascript to read the user's cookie to set the `body` class name.

I'm guessing setting the `src` of images is a bit more involved, but that should be fine.

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning can this be closed due to your changes in #1171?

**Comment by brockfanning** [comment]
Yes, I'll close this now.
